tests/kafkatest/services/performance/consumer_performance.py

```diff
@@ -70,14 +70,16 @@ class ConsumerPerformanceService(PerformanceService):
             "collect_default": True}
     }
 
-    def __init__(self, context, num_nodes, kafka, topic, messages, version=DEV_BRANCH, new_consumer=True, settings={}):
+    def __init__(self, context, num_nodes, kafka, topic, messages, version=DEV_BRANCH, new_consumer=True, settings={}, consumer_config={}, timeout_sec=None):
```

Comment:
Maybe a more descriptive name would help for `timeout_sec`. I was confused because I thought this was going to be used as the `--timeout` value for the performance command.

Reply:
Yes, `timeout_sec` implements the same behavior as `--timeout` option. The only reason we can't use `--timeout` directly is because it is not available in older versions of `ConsumerPerformance` tool.

Reply:
Yes, that make sense, but is the behavior the same? The `--timeout` option is the maximum time the performance tool will await for new records. Does the timeout option to `ssh_capture` do the same thing?

Reply:
Yeah, this is somewhat different in the sense that this is a timeout for the ssh command itself.